dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePlugin.kt

```diff
@@ -397,10 +400,16 @@ class DdAndroidGradlePlugin @Inject constructor(
             )
             return
         }
-
+        val composeInstrumentation = ddExtension.composeInstrumentation
         project.tasks.withType(KotlinCompile::class.java).configureEach { task ->
             task.kotlinOptions.freeCompilerArgs += listOf(
-                "-Xplugin=$pluginJarPath"
+                "-Xplugin=$pluginJarPath",
+                "-P",
+                "plugin:$DATADOG_PLUGIN_ID:$TRACK_VIEWS=${composeInstrumentation.trackViews.name}",
+                "-P",
+                "plugin:$DATADOG_PLUGIN_ID:$TRACK_ACTIONS=${composeInstrumentation.trackActions.name}",
+                "-P",
+                "plugin:$DATADOG_PLUGIN_ID:$RECORD_IMAGES=${composeInstrumentation.recordImages.name}"
```

Comment:
probably there is a confusion here and we should use ID of the `DatadogKotlinCompilerPlugin` (which in turn should be also different from the ID used by `DdAndroidGradlePlugin`).